docs/VLSI/Sky130-OpenROAD-Tutorial.rst

```diff
@@ -0,0 +1,220 @@
+.. _sky130-openroad-tutorial:
+
+Sky130 + OpenROAD Tutorial
+==========================
+The ``vlsi`` folder of this repository contains an example Hammer flow with the TinyRocketConfig from Chipyard. This example tutorial uses the built-in Sky130 technology plugin and OpenROAD tool plugin.
+
+Project Structure
+-----------------
+
+This example gives a suggested file structure and build system. The ``vlsi/`` folder will eventually contain the following files and folders:
+
+* ``Makefile``, ``sim.mk``, ``power.mk``
+
+  * Integration of Hammer's build system into Chipyard and abstracts away some Hammer commands.
+
+* ``build``
+
+  * Hammer output directory. Can be changed with the ``OBJ_DIR`` variable.
+  * Will contain subdirectories such as ``syn-rundir`` and ``par-rundir`` and the ``inputs.yml`` denoting the top module and input Verilog files.
+
+* ``example-vlsi-sky130``
+
+  * Entry point to Hammer. Contains example placeholders for hooks.
+
+* ``example-sky130.yml``, ``example-openroad.yml``
+
+  * Hammer IR for this tutorial.
+
+* ``example-design.yml``, ``example-asap7.yml``, ``example-nangate45.yml``, ``example-tech.yml``
```

Comment:
We might as well remove nangate45, since OpenROAD plugin updates have broken it and no tutorial targeting it has ever existed, anyway.